quiz_app_questions.json

```
@@ -35,9 +35,9 @@
 		"answer": "Motherboard"
 	},
 	{
-		"question": "Which of these is not a peripheral, in computer terms?",
-		"options": [ "Keyboard", "Mouse", "Motherboard", "Monitor" ],
-		"answer": "Motherboard"
+		"question": "The first alpha version of Flutter was released in?",
```

Womp! Sorry for the multiple updates. Grammar-wise and to match the rest of the questions, it would be better to reword this to:

```suggestion
		"question": "When was the first alpha version of Flutter released?",
```

---

Yes, this is better version. Thankyou.